.goreleaser.yml

```diff
@@ -13,14 +13,15 @@ builds:
     ldflags:
       - -s -w -X github.com/skyscanner/turbolift/cmd.version={{.Version}} -X github.com/skyscanner/turbolift/cmd.commit={{.Commit}} -X github.com/skyscanner/turbolift/cmd.date={{.Date}}
 archives:
-  - replacements:
-      darwin: Darwin
-      linux: Linux
-      windows: Windows
-      386: i386
-      amd64: x86_64
+  - name_template: >-
+      {{- .ProjectName }}_
+      {{- title .Os }}_
+      {{- if eq .Arch "amd64" }}x86_64
+      {{- else if eq .Arch "386" }}i386
+      {{- else }}{{ .Arch }}{{ end }}
+      {{- if .Arm }}v{{ .Arm }}{{ end -}}
```

Comment:
Does the trailing `-}}` trim all empty spaces and newlines afterwards?

Reply:
yeah that's correct! https://developer.hashicorp.com/nomad/tutorials/templates/go-template-syntax#delimiters